src/Umbraco.Core/Configuration/Models/GlobalSettings.cs

```diff
@@ -46,13 +49,17 @@ internal const string
 
         public bool DisableElectionForSingleServer { get; set; } = false;
 
+        public DatabaseServerRegistrarSettings DatabaseServerRegistrar { get; set; } = new DatabaseServerRegistrarSettings();
+
+        public DatabaseServerMessengerSettings DatabaseServerMessenger { get; set; } = new DatabaseServerMessengerSettings();
+
         public string RegisterType { get; set; } = string.Empty;
 
         public string DatabaseFactoryServerVersion { get; set; } = string.Empty;
 
         public string MainDomLock { get; set; } = string.Empty;
 
-        public string NoNodesViewPath { get; set; } = "~/config/splashes/NoNodes.cshtml";
+        public string NoNodesViewPath { get; set; } = "~/umbraco/UmbracoWebsite/NoNodes.cshtml";
```

Comment:
@bergmania we will need to fix these hard coded ~/umbraco paths, they cannot be hard coded like this since the umbraco path comes from the UmbracoPath value. We can change later (not in this PR)